lib/tailwind_formatter.ex

```diff
@@ -125,11 +125,14 @@ defmodule TailwindFormatter do
 
   defp placeholder?(class), do: String.contains?(class, @placeholder)
   defp variant?(class), do: String.contains?(class, ":")
+  defp prose?(class), do: String.contains?(class, "prose")
 
   defp class_position(class),
     do: if(placeholder?(class), do: -1_000_000, else: Map.get(Order.classes(), class, -1))
 
-  defp variant_position(variant), do: Map.get(Order.variants(), variant, -1)
+  # prose variant order matters, thus push to front
+  defp variant_position(variant),
+    do: if(prose?(variant), do: 0, else: Map.get(Order.variants(), variant, -1))
```

Comment:
This is due to #8 – `prose:` modifiers need to be the final variant in order to apply correctly. In addition, the ordering of `prose` classes can cause issues. 

If this still causes issues, then we can either

1. Do some post-processing on the  `variants.txt` to push `prose-*` modifiers to the front
2. Just not call `sort_variant_chains` when `prose` is found as a substring